melange-atdgen-codec-runtime.opam

```diff
@@ -13,7 +13,7 @@ depends: [
   "ocaml"
   "melange" {>= "3.0.0"}
   "atd"
-  "atdgen"
+  "atdgen" {>= "2.16.0"}
```

Comment:
Does it make sense to bump the version here?

Reply:
Yes. if we don't do this, the opam repo checks will fail.